src/ui/fx-control.js

```diff
@@ -31,29 +34,45 @@ class FxControl extends XfAbstractControl {
             }
         `;
 
-    /*
-        const controlHtml = `
-            <slot></slot>
-            <fx-setvalue id="setvalue" ref="${this.ref}"></fx-setvalue>
-        `;
-*/
+    if (this.hasAttribute('uri')) {
+      const subForm = this.ownerDocument.querySelector(this.getAttribute('uri'));
+
+      this.subForm = this.shadowRoot.appendChild(this.ownerDocument.createElement('fx-fore'));
+      this.subForm.appendChild(subForm.content.cloneNode(true));
```

Comment:
not sure about this one - shouldn't the content of the template be a 'fx-fore' element?